src/FormInput/index.tsx

```diff
@@ -89,7 +89,19 @@ const FormInputValidation: React.FunctionComponent<IFormInputValidationProps> =
   errors = [],
   tooltipProps,
 }) => {
-  const errs = useValidateSchema(schema, value, { abortEarly: false }).concat(errors);
+  const { errors: validationErrors, isStale } = useValidateSchema(schema, value, { abortEarly: false });
+  const errs = [...validationErrors, ...errors];
+
+  if (isStale && errs.length) {
+    return (
+      <Tooltip content="Validating..." position={Position.BOTTOM_RIGHT}>
+        <div tabIndex={-1} style={{ height: iconHeight[size] }} className="mr-2">
+          <Icon icon="outdated" iconSize={size === 'small' ? 12 : Icon.SIZE_STANDARD} intent="danger" />
```

Comment:
What do you think about using a red [spinner](https://blueprintjs.com/docs/#core/components/spinner) here while we're validating instead of the outdated icon? I'm afraid the outdated icon might be confusing

Reply:
1. I agree that the spinner represents better the operation in progress.
2. For async validation I think the spinner should always appear, even if the current input is valid